autoload/clang_format.vim

```diff
@@ -64,7 +64,7 @@ endfunction
 
 function! s:make_style_options() abort
     let extra_options = s:build_extra_options()
-    return printf("'{BasedOnStyle: %s, IndentWidth: %d, UseTab: %s%s}'",
+    return printf("\"{BasedOnStyle: %s, IndentWidth: %d, UseTab: %s%s}\"",
```

Comment:
Let me ask a quetion here. In Windows shell, should we need to escape backslash because of double quote on cmd.exe?

I used single quotes here to avoid escaping backslash in command line arguments.